GW2EIBuilders/HtmlModels/LogDataDto.cs

```diff
@@ -60,12 +60,16 @@ internal class LogDataDto
     public BarrierStatsExtension? BarrierStatsExtension;
     // meta data
     public string EncounterDuration;
+    public string EvtcRecordingDuration;
```

Comment:
Any reason this was not added to the official JSON?

Reply:
Didn't think about it, not sure if it's needed?

Reply:
I guess it could be computed from TimeStart and TimeEnd if desired. Fair enough.